moveit_setup_assistant/templates/moveit_config_pkg_template/launch/moveit_rviz.launch

```diff
@@ -10,7 +10,6 @@
 
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
-    <rosparam command="load" file="$(find [GENERATED_PACKAGE_NAME])/config/kinematics.yaml"/>
```

Comment:
I guess, removing this line makes rviz not find the kinematics parameters anymore.

Reply:
planning_context is included which loads the parameters. Suggested by @v4hn , tested and still works fine without that line

Reply:
I don't see `planning_context.launch` being included here. It is, indeed, for `run_benchmark_ompl.launch`.

Reply:
Agreed, but that file does not load other parameters either ... so my understanding was that it is not meant to be used standalone but rather as an include by other launchfiles?

Reply:
I think you can use `moveit_rviz.launch` "standalone" to just load rviz, e.g. in a separate terminal. However, you are right that this usually assumes that `planning_context.launch` was launched somewhere else.
What I don't really understand right now, is why we used separate kinematics parameters for rviz in the past, but remove them now. Maybe @v4hn can help?

Reply:
For reasons we addressed over two years ago:

https://github.com/ros-planning/moveit/pull/701

Reply:
Thanks for this link.